functions-and-scope.js

```diff
@@ -78,6 +150,22 @@ const grades = [9, 8, 5, 7, 7, 4, 9, 8, 8, 3, 6, 8, 5, 6];
 // Zorg ervoor dat jouw functie ook werkt als we een andere array willen checken, zoals bijvoorbeeld: [6, 4, 5] of [8, 9, 4, 6, 10].
 // Log het antwoord in de terminal.
 
+console.log("Opdracht 3b:");
+
+function highestGrade(num) {
+    let large = 0;
+    for (let i = 0; i < num.length; i++){
+        if (large < num[i]) {
+            large = Math.max(num[i]);
```

Comment:
Math.max met maar één parameter geeft altijd dezelfde parameter weer terug. Wat je zou kunnen doen is de hele array met cijfers erin gooien: `return Math.max(...num);`